### DISCUSSION TITLE
I want my tests which are in different namespace and files to access and  write information into it.

### DISCUSSION THREAD

**Discussion by kabdul-incomm** [nunit/nunit#4616] [Q&A] [answered]
I want to create a file before my tests run and I want my tests which are in different namespace and files to access a notepad file and write information into it. Could you please help me here.

NOte: Tests are in a same dll but different namespace.

**Comment by OsirisTerje** [comment, marked as answer]
You can try using a global setupfixture, see https://docs.nunit.org/articles/nunit/writing-tests/attributes/setupfixture.html?q=setupfixture
and note 

> A SetUpFixture outside of any namespace provides SetUp and TearDown for the entire assembly.
